integrations/serverless-ossm-setup.adoc

```diff
@@ -24,10 +35,31 @@ To complete and verify these procedures in your deployment, you need either a ce
 
 * If you want to use any domain name, including those which are not subdomains of the default {ocp-product-title} cluster domain, you must set up domain mapping for those domains. For more information, see the {ServerlessProductName} documentation about xref:../knative-serving/config-custom-domains/create-domain-mapping.adoc#serverless-create-domain-mapping_create-domain-mapping[Creating a custom domain mapping].
 
+[IMPORTANT]
+====
+{ServerlessProductName} only supports the use of {SMProductName} functionality that is explicitly documented in this guide, and does not support other undocumented features.
+
+Using {ServerlessProductShortName} 1.31 with {SMProductShortName} is only supported with {SMProductShortName} version 2.2 or later. See the "Red Hat OpenShift Serverless Supported Configurations" page for details and information on versions other than 1.31.
```

Comment:
Recast to give the reason for the link first. Also, use IBM standard language: "For information about versions other than 1.31, see the "Red Hat OpenShift Serverless Supported Configurations" page ."